src/components/Header/Header.js

```diff
@@ -1,13 +1,13 @@
 "use client";
 
-import Link from 'next/link';
+import { Link } from "@/lib/i18n"
 import {useState} from 'react';
-import HeadStyles from './Header.module.css';
+import HeadStyles from './Header.module.scss';
 import logo from '@/assets/images/logo.png?static';
 import Image from 'next/image';
-import BigYellowButton from "@/components/BigYellowButton";
-import SearchField from "@/components/SearchField";
-import SmallRoundButton from "@/components/SmallRoundButton";
+import BigYellowButton from "@/components/BigButton/BigButton";
+import SearchField from "@/components/SearchField/SearchField";
+import SmallRoundButton from "@/components/SmallRoundButton/SmallRoundButton";
```

Comment:
Добавить index.js во все компоненты, чтобы пути выглядели так 
```suggestion
import BigYellowButton from "@/components/BigButton";
import SearchField from "@/components/SearchField";
import SmallRoundButton from "@/components/SmallRoundButton";
```